app/e2e/auth.setup.ts

```diff
@@ -0,0 +1,29 @@
+import { test as setup, expect } from "@playwright/test";
+
+const authMethod = process.env.PACKIT_E2E_AUTH_METHOD || "basic";
+const basicUser = process.env.PACKIT_E2E_BASiC_USER || "resideUser@resideAdmin.ic.ac.uk";
```

Comment:
```suggestion
const basicUser = process.env.PACKIT_E2E_BASIC_USER || "resideUser@resideAdmin.ic.ac.uk";
```

there's a typo there on the I being lowercase instead of uppercase